contact.html

```diff
@@ -0,0 +1,211 @@
+      crossorigin="anonymous"></script>
+    <!-- Link to JS Popper -->
+    <script
+      src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.5.4/dist/umd/popper.min.js"
+      integrity="sha384-q2kxQ16AaE6UbzuKqyBE9/u/KzioAlnx2maXQHiDX9d4/zp8Ok3f+M7DPm+Ib6IU"
+      crossorigin="anonymous"></script>
+    <!-- Link for Boostrap JS -->
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/js/bootstrap.min.js"
+      integrity="sha384-pQQkAEnwaBkjpqZ8RU1fF1AKtTcHJwFl3pblpTlHXybJjHpMYo79HY3hIi4NKxyj"
+      crossorigin="anonymous"></script>
+    <!-- Font Awesome -->
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.11.2/css/all.css"/>
+    <!-- Google Fonts Exo 2 -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Exo+2&display=swap" rel="stylesheet">
+    <!-- Link to CSS file -->
+    <link rel="stylesheet" href="./css/main.css" />
+    <title>Bootstrap - Responsive grid project</title>
+  </head>
+
+  <body class="portfolio-body mt-5">
+
+    <header>
+      <!-- Navigation bar  -->
+      <nav
+        class="navbar navbar-expand-lg navbar-light align-items-start"
+      >
+        <div class="container-fluid d-inline-flex fixed-top bg-white">
+          <div class="row">
+            <div
+              class="col-xs-12 col-sm-8 col-md-6 col-lg-4 text-nowrap text-center"
+            >
+              <a class="navbar-brand" href="#"
+                ><span style="font-size: 20px; color: rgb(107, 10, 136)"
+                  >&#9639;</span
+                >
+                <b>Laura Menendez</b></a
+              >
+              <p>TRAINEE DEVELOPER</p>
+            </div>
+          </div>
+          <button
+            class="navbar-toggler"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarNavAltMarkup"
+            aria-controls="navbarNavAltMarkup"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div
+            class="collapse navbar-collapse justify-content-center text-center"
+            id="navbarNavAltMarkup"
+          >
+            <div class="navbar-nav">
+              <a class="nav-link active" aria-current="page" href="index.html"
+                >About</a
+              >
+              <a class="nav-link" href="resume.html">Resume</a>
+              <a class="nav-link" href="portfolio.html">Portfolio</a>
+              <a class="nav-link" href="contact.html">Contact</a>
+            </div>
+          </div>
+        </div>
+      </nav>
+    </header>
+
+<main>
+    <div class="container contact shadow p-sm-3 mt-5">
+      <form action="action_page.php" method="post">
+        <div class="row">
+          <div class="col-25">
```

Comment:
You should bootstrap's class for this (instead of your own CSS class) as it has an excellent capability to design your form layout